lib/suspenders/app_builder.rb

```diff
@@ -130,6 +134,10 @@ def configure_smtp
     def enable_rack_canonical_host
       config = <<-RUBY
 
+  if ENV["HEROKU_APP_NAME"].include?("staging-pr-")
```

Comment:
In my testing of this on a side project I ran into an issue where the ENV var `HEROKU_APP_NAME` was not set on the main staging server and only set on the review apps. Not sure if that was just something I did wrong but I ended up changing this to:

```ruby
if ENV.fetch("HEROKU_APP_NAME", "").include?("staging-pr-")
# ...
end
```